__tests__/nc-news.test.js

```diff
@@ -209,4 +209,21 @@ describe("ncNews API tests", () => {
         expect(updatedArticle).toBe("The article_id provided does not exist");
       });
   });
+  test("DELETE: 204 - /api/comments/:comment_id - Delete a comment with the given comment_id", () => {
+    return request(app)
+      .delete("/api/comments/16")
+      .expect(204)
+      .then(() => {
+        // Nothing was sent, so I check the GET endpoint and expect 404-NotFound
+        return request(app).get("/api/comments/16").expect(404);
+      });
+  });
+  test("DELETE: 404 - /api/comments/:comment_id - try to Delete a comment, but comment_id does not exist", () => {
+    return request(app)
+      .delete("/api/comments/888")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Comment with id 888 does not exist.");
+      });
+  });
```

Comment:
You need to add in another test for a - Status 400, invalid ID, e.g "not-an-id" 💡